executor/studentapi.pyx

```diff
@@ -164,6 +164,28 @@ cdef class Robot:
         return False
 
 
+    cpdef void log(self, str key, value) except *:
```

Comment:
it tells C to check for a Python exception after this function is called. otherwise, any python errors will be swallowed. due to how C doesn't actually have exceptions